qiita_pet/handlers/qiita_redbiom.py

```diff
@@ -27,9 +29,18 @@ class RedbiomPublicSearch(BaseHandler):
     def get(self, search):
         # making sure that if someone from a portal forces entry to this URI
         # we go to the main portal
+        try:
+            timestamps = redbiom.admin.get_timestamps()
+        except (rHTTPError):
+            timestamps = []
+
+        if timestamps:
+            latests_release = timestamps[0]
```

Comment:
This is minor but the variable name should be `latest_releases` right? If you agree then make sure to also update the template.

Reply:
Happy to change but note that is only taking one by `[0]`

Reply:
Gotcha, then `latest_release` should work well.